x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/inference_pipeline_card.test.tsx

```diff
@@ -67,20 +67,21 @@ describe('InferencePipelineCard', () => {
     const badge = wrapper.find(MLModelTypeBadge).render();
     expect(badge.text()).toBe('ner');
   });
-  it('renders fix button when model not deployed', () => {
+  it('popover renders fix button when model not deployed', () => {
```

Comment:
Minor: consider extracting the popover-related test cases into its own `describe('TrainedModelHealthPopover', () => ...` block, since it's now a separate component.